README.md

```diff
@@ -1,295 +1,314 @@
 
-```ruby
-require 'net/http'
-require "learnosity/sdk/request/init"
+Note that these manual installation methods are for development and testing only.
+For production use, you should install the SDK using the Pip package manager for Ruby, as described above.
 
+[(Back to top)](#table-of-contents)
 
-security_packet = {
-	# XXX: This is a Learnosity Demos consumer; replace it with your own consumer key
-	'consumer_key'   => 'yis0TYCu7U9V4o7M',
-	'domain'         => 'localhost'
-}
-# XXX: The consumer secret should be in a properly secured credential store, and *NEVER* checked in in revision control
-consumer_secret = '74c5fd430cf1242a527f6223aebd42d30464be22'
-data_request = { 'limit' => 50 }
+## Quick start guide
+Let's take a look at a simple example of the SDK in action. In this example, we'll load an assessment into the browser.
 
-init = Learnosity::Sdk::Request::Init.new(
-        'data',
-        security_packet,
-        consumer_secret,
-        data_request,
-	'get'
-)
+### **Start up your web server and view the standalone assessment example**
+To start up your Ruby web server, first find the following folder location under the SDK. Change directory ('cd') to this location on the command line.
 
-request = init.generate
+    learnosity-sdk-ruby/examples/lrn-sdk-rails/
 
-Net::HTTP.post_form URI('https://data.learnosity.com/v1/itembank/items'), request
-```
+To start, run this command from that folder:
 
-### Recursive Queries
+    rails server
 
-tl;dr: not currently implemented
+From this point on, we'll assume that your web server is available at this local address (it will report the port being used when you launch it, by default it's port 3000): 
 
-Some requests are paginated to the `limit` passed in the request, or some
-server-side default. Responses to those requests contain a `next` parameter in
-their `meta` property, which can be placed in the next request to access another
-page of data.
+http://localhost:3000/index/index
 
-For the time being, you can iterate through pages by looping over the
-`Init#new`/`Init#generate`/`Net::HTTP#post_form`, updating the `next` attribute
-in the request.
+The page will load. This is a basic example of an assessment loaded into a web page with Learnosity's assessment player. You can interact with this demo assessment to try out the various Question types.
 
-```ruby
-response = JSON.parse(res.body)
-if ( !response['meta']['next'].nil? \
-		and !response['meta']['records'].nil? and response['meta']['records'] > 0)
-	data_request['next'] = response['meta']['next']
-end
-```
+<img width="50%" height="50%" src="docs/images/image-quickstart-examples-assessment.png">
 
-This will `require 'json'` to be able to parse the response.
+[(Back to top)](#table-of-contents)
 
-See `examples/simple/init_data.rb` for an example.
+### **How it works**
+Let's walk through the code for this standalone assessment example. The source files are included under the [/example/lrn-sdk-rails/] folder.
```

Comment:
That link is not rendered correctly.

Reply:
Fixed. https://github.com/Learnosity/learnosity-sdk-ruby/commit/f9d2964750ec3dd30a51e1b8e7b0af431a9e15c5